grid/address/ip-geolocation/maps/ipstack.test.ts

```diff
@@ -0,0 +1,30 @@
+import { RecordingDefinitions } from '@superfaceai/testing';
+
+import { ipGeolocationTest } from './ip-geocoding';
+
+ipGeolocationTest('ipstack', {
+  //Testing library is not processing recordings because integration parameter is used to replace HTTP scheme which is not supported by testing lib
+  processRecordings: false,
+  beforeRecordingSave: (recordings: RecordingDefinitions) => {
+    const IPSTACK_API_KEY_REDACTED_VALUE = 'IPSTACK_API_KEY_REDACTED';
```

Comment:
This value should be better pulled out to the outer scope.

Reply:
fixed